labs/lab12/sol/lin/1-mini/README.md

```diff
@@ -0,0 +1,17 @@
+# Indicații suplimentare
+
+## Exerciţiul 1
+* `ls -l /dev` va lista toate device node-urile.
+* Exemplu:
+
+```
+brw-rw---T 1 root disk 8, 1 May 14 19:50 /dev/sda1
+b - block device, 8 - major, 1 - minor
+```
+
+## Exerciţiul 3
+* trebuie să includeţi `<sys/mount.h>`
+
+## Exerciţiul 6
+* man opendir
+* trebuie incluse `<sys/types.h>` şi `<dirent.h>`
```

Comment:
# Additional Hints

## Task 1
* `ls -l /dev` will list all node devices. 
* Check the snippet below:

```
# b - block device, 8 - major, 1 - minor
brw-rw---T 1 root disk 8, 1 May 14 19:50 /dev/sda1
```

## Task 3
* Include `<sys/mount.h>`.

## Task 6
* `man opendir`
* Include `<sys/types.h>` and `<dirent.h>`.

Reply:
I would translate this whole README in English.